news-app/pages/index.js

```diff
@@ -0,0 +1,49 @@
+import Link from "next/link";
+
+export default function Home({ value }) {
+  return (
+    <div className="flex justify-center items-center flex-col">
+      <h3 className="text-secondary text-2xl font-raleway font-bold uppercase tracking-wide mb-10 md:text-lg">
+        Get Top <span className="text-danger">News</span> Quickly
+      </h3>
+      {value.map((news) => {
+        return (
+          <Link href={news.url}>
```

Comment:
There should be an anchor tag immediately after the Next.js `Link` component [unless the child is a functional component](https://nextjs.org/docs/api-reference/next/link#if-the-child-is-a-functional-component).

Reply:
For Lists or Arrays, you need to use [keys](https://reactjs.org/docs/lists-and-keys.html#keys).